cmd/flag/open_stream_flags.go

```diff
@@ -17,15 +17,30 @@ package flag
 
 import (
 	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	defaultEnableAutoClose = false
+	defaultAutoCloseDelay  = 5 * time.Second
 )
 
 type OpenStreamFlag struct {
-	StreamId string
+	EnableAutoClose bool
+	AutoCloseDelay  time.Duration
+	AutoCloseTime   string
+	StreamId        string
 }
 
 // Add a flag to the command.
 func (f *OpenStreamFlag) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.StreamId, "stream-id", "r", "", "The StreamId to resume.")
+	cmd.Flags().BoolVarP(&f.EnableAutoClose, "enable-auto-close", "", defaultEnableAutoClose,
+		"When set to true, the stream will close after a specified auto close time.")
+	cmd.Flags().DurationVarP(&f.AutoCloseDelay, "auto-close-delay", "", defaultAutoCloseDelay,
+		"The duration to wait before ending the stream with no more data incoming. Valid time units are \"s\", \"m\". Ex 1m30s. Range 1s to 10m")
+	cmd.Flags().StringVarP(&f.AutoCloseTime, "auto-close-time", "", "",
+		"The datetime (in UTC) after which auto-closing will be enabled. Format 2006-01-02 15:04:05")
```

Comment:
```suggestion
		"The datetime (UTC) after which auto-closing will be enabled. Format 2006-01-02 15:04:05")
```

Sorry for nitpicking, just trying to match documentation of other commands 😆